src/Avalonia.Visuals/Rendering/ICustomSimpleHitTest.cs

```diff
@@ -25,7 +25,7 @@ public interface ICustomHitTest : ICustomSimpleHitTest
     public static class CustomSimpleHitTestExtensions
     {
         public static bool HitTestCustom(this IVisual visual, Point point)
-            => (visual as ICustomSimpleHitTest)?.HitTest(point) ?? visual.TransformedBounds?.Contains(point) == true;
+            => (visual as ICustomSimpleHitTest)?.HitTest(point) ?? false;
```

Comment:
In the absence of `ICustomSimpleHitTest` we still need to check against the control bounds I think?

Reply:
Reverted but switched to using Bounds instead of TransformedBounds